tests/flow/test_evaluation_imperative.py

```diff
@@ -580,3 +580,15 @@ def invoke(self, query: dict, **options) -> str:
         )
         pred.finish()
         ev.finish()
+
+
+def test_evaluation_logger_with_custom_attributes(client):
+    ev = weave.EvaluationLogger(eval_attributes={"custom_attribute": "value"})
+    pred = ev.log_prediction(inputs={"a": 1, "b": 2}, output=3)
+    ev.log_summary(auto_summarize=False)
+    ev.finish()
+    client.flush()
+
+    calls = client.get_calls()
+    for call in calls:
+        assert call.attributes["custom_attribute"] == "value"
```

Comment:
Is this what we should expect?  Why do you want to apply a custom attribute to all calls instead of just the root call?